test/cloudtest/test_storage.py

```diff
@@ -226,10 +254,12 @@ def get_num_workers(mz: MaterializeApplication) -> str:
     wait(
         condition="condition=Ready",
         resource="pod",
-        label="storage.environmentd.materialize.cloud/size=16",
+        label="cluster.environmentd.materialize.cloud/workers=4",
     )
 
-    assert get_num_workers(mz) == "'16'"
+    assert get_cluster_workers(mz, cluster_id) == "'4 4 4 4'"
+    assert get_cluster_scale(mz, cluster_id) == "'4 4 4 4'"
+    assert get_cluster_size(mz, cluster_id) == "'4-4 4-4 4-4 4-4'"
```

Comment:
You could also print out the actual sizes in these assertions if they fail , using `assert get_cluster_size(mz, cluster_id) == "'4-4 4-4 4-4 4-4'", f"Actual cluster size: {get_cluster_size(mz, cluster_id)}"`

Reply:
How is it written?

Reply:
Apparently pytest provides a better `assert` implementation:

```
import pytest

def f():
   return 1

def test():
    assert f() == 2
```

Reply:
Ok, cool. I was wondering about that before when I saw the better assert output. Hadn't used pytest before Mz.